### ISSUE TITLE
Linking

### ISSUE THREAD

**Issue by gedw99** [gomatcha/matcha#40]
I am trying to link to another objective c project that supports webrtc native.

https://github.com/ISBX/apprtc-ios/blob/master/README.md

This will allow the GUI aspect to be done with matcha and the logic in golang. Now I am trying to work out how to design the protobuf integration.

If you have any thought on it.
I am hoping that we can get set of plugin code just like react and native script has.

**Comment by overcyn** [comment]
Yeah, to be honest you don't actually need to use protobuf, `view.Model.NativeViewState` takes in a byte slice. Using JSON or something might make getting started easier.
